.github/workflows/publish.yml

```
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    if: github.repository == 'astropy/pytest-astropy-header'
+
+    steps:
+    - uses: actions/checkout@master
```

I missed this over at pytest-doctestplus . I think we shouldn't use `master`, especially given their README recommends `v2`.
```suggestion
    - uses: actions/checkout@v2
```